files/sbin/entrypoint

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh -e
+
+chown -R uwsgi:uwsgi /var/opt/taiga-back
+
+if [ "$#" -ne 0 ]; then
+	exec "$@"
+fi
+
+./manage.py check --deploy
+./manage.py collectstatic --no-input
```

Comment:
Wouldn't it be more efficient to use `collectstatic --link`?

Reply:
From storage efficiency viewpoint, yes. But not from the configuration one. The problem is, static assets can come from a huge number of different packages, so whitelisting them all in the server configuration can be impractical. We could solve both problems with hard links, but django-admin doesn't provide such option.